### DISCUSSION TITLE
How to change the color of the placeholder?

### DISCUSSION THREAD

**Discussion by cfclaudio** [codex-team/editor.js#1917] [Q&A]
I have tried adding `.ce-paragraph[data-placeholder]:empty::before {
  color: red;
}
` to my stylesheet but it does not work.

**Comment by neSpecc** [comment]
could you show a reproducible demo page?

**Comment by victor-david** [comment]
Try adding **!important** to the def. That's what worked for me. Without it, no.
~~~css
.ce-paragraph[data-placeholder]:empty::before { color: red !important }
~~~

**Reply by pranavmappoli** [reply]
@victor-david i dont know, its not working for me now
@neSpecc Could you please help

`<div class="ce-paragraph cdx-block" contenteditable="true" data-placeholder-active="Provide a clear and helpful answer.." data-empty="true"></div>`

Tried all below
```
.codex-editor [data-placeholder-active]:empty::before,
.codex-editor [data-placeholder-active][data-empty="true"]::before {
  background-color: red !important;
}
.ce-paragraph[data-placeholder]:empty::before { color: red !important }
```
